src/psalter.php

```diff
@@ -31,6 +32,7 @@
     'plugin:', 'issues:', 'list-supported-issues', 'php-version:', 'dry-run', 'safe-types',
     'find-unused-code', 'threads:', 'codeowner:',
     'allow-backwards-incompatible-changes:',
+    'doc-block-add-new-line-before-return:',
```

Comment:
Could you change this command to `add-newline-between-docblock-annotations` please?

And then slightly change your implementation, because now you wouldn't need the explicit check for `'return'` – just this flag.

I'm thinking that anyone who doesn't want newlines before `@return`  _also_ doesn't want newlines before `@throws`

Reply:
Do you ever sleep? ;) 

Yes, totally makes sense and I will do so tomorrow. Just had one too many beers today to complete it now.

Reply:
Yes, I sleep in New York!